lib/usd/ui/TreeModel.cpp

```diff
@@ -73,6 +73,7 @@ void resetVariantToPrimSelection(TreeItem* variantItem)
 	}
 
 	variantItem->setData(qtVarNames, ItemDelegate::kVariantSelectionRole);
+	variantItem->resetVariantSelectionModified();
```

Comment:
Already had this function to reset the variant selection, but also need to tell the TreeItem that this has happened because we use its state when counting.